frontend/src/pages/LoginRegister.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState, useEffect } from "react";
+import LoginRegisterForm from "components/LoginRegisterForm";
+import styled from "styled-components";
+
+const LoginRegister = ({ API_URL }) => {
+  const [loginOrRegister, setLoginOrRegister] = useState("login");
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleFormSubmit = (event, state) => {
+    switch (state) {
+      case "login":
+        console.log("login");
+        fetch(`${API_URL}/login`, {
+          method: "POST",
+          headers: { "Content-Type": "application/json" },
+          body: JSON.stringify({ email, password }),
+        })
+          .then((response) => response.json())
+          .then((data) => {
+            if (data.success) {
+            localStorage.setItem("token", data.response.accessToken);
+            window.location.reload(); } else { alert("Login error!") }
+          })
+          .catch((error) => console.log(error))
+
+        break;
```

Comment:
I don't remember seeing 'break' statement during the bootcamp before, nice touch!